Programs/_freeze_module.c

```diff
@@ -1,5 +1,10 @@
 /* This is built as a stand-alone executable by the Makefile, and helps turn
-   Lib/importlib/_bootstrap.py into a frozen module in Python/importlib.h
+   modules into frozen modules (like Lib/importlib/_bootstrap.py
+   into Python/importlib.h)
+
+   This is used directly by Tools/scripts/freeze_modules.py, and indirectly by "make regen-frozen-*".
```

Comment:
Is this line out of date? IIUC it's now only used by targets put in the Makefile by freeze_modules.py? And there's no "make regen-frozen-*" -- there's only "make regen-frozen".

Reply:
Yeah, I'll fix it.